cmd/helper-reset-password/main.go

```diff
@@ -60,14 +75,19 @@ func main() {
 		}
 	}
 
+	// If password is used for docker extension. It won't return an error if passwords are same.
+	if err := cryptoService.CompareHashAndData(user.Password, "K7yJPP5qNK4hf1QsRnfV"); err == nil {
+		log.Fatalf("Database from a Docker Desktop Portainer instance detected - exiting without resetting")
+	}
+
```

Comment:
Nooo, Do not do this.   Never put the password into the source.  Also it'll trigger github security scan thing.  Please remove this from git history with commit -amend.   

What you should do is literally read the hash value and compare it with the literal hash.  Do not take the password and compute the hash and compare them.  Just compare the two hashes directly.